doc/api.yml

```diff
@@ -1400,6 +1640,16 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/StudentExamGrade'
+    CreatePointing:
```

Comment:
CreateMovement or CreateAttendanceMovement or something like this.

Reply:
checked, change to CreateAttendanceMovement